src/apps/content/templates/support/terms_and_conditions.html

```diff
@@ -1,21 +1,23 @@
 {% extends "base.html" %}
+{% load static %}
```

Comment:
isn't it better to include {% load static %} to the base.html
so we wouldn't add this line in every other template

Reply:
i think we need static files not so often inside the content block, what other use cases are you thinking off?

Reply:
fixed